### DISCUSSION TITLE
Vercel Server has been blocked by Telco

### DISCUSSION THREAD

**Discussion by namanhvn** [vercel/community#4070] [Help]
### Summary

Dear Vercel Team,

I'm deploying my website with paid plan. Our website is serving Vietnamese customer only. The problem is:

1. I can visit the website normally, however, sometimes, it's unreachable with Viettel Network.
2. I tried to open a VPN to Singapore, it worked.
3. This issue does not happened with other telco network.

I'm thinking about a workaround way of moving out from Vercel to a standalone server, however, I love Vercel.
I suggest that you should have different regions and servers worldwide so we can select the best server to use and avoid telco's blocking.

### Steps to Reproduce

It happened sometimes.

**Comment by amyegan** [comment]
Thanks for letting us know this is happening. Our team is aware of the problem and is working to unblock Vercel's IP address. 🙇
